tendermint/tests/integration.rs

```diff
@@ -34,7 +34,7 @@ mod rpc {
     async fn abci_info() {
         let abci_info = localhost_rpc_client().abci_info().await.unwrap();
 
-        assert_eq!(&abci_info.version, "0.16.2");
+        assert_eq!(&abci_info.version, "0.17.0");
```

Comment:
Good, this will also close: https://github.com/informalsystems/tendermint-rs/issues/249